Algorithms/Problems/CoinChange/GreedyAlgorithm.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq.Expressions;
+using System.Security.Cryptography;
+using System.Text;
+
+public class GreedyAlgorithm
+
+    /*
+    // Info
+    //
+A greedy algorithm is the one that always chooses the best solution at the time, with no regard for how that choice will affect future choices. Here, we will discuss how to use Greedy algorithm to making coin changes.
+It has been proven that an optimal solution for coin changing can always be found using the current American denominations of coins
+For an example, Let’s say you buy some items at the store and the change from your purchase is 63 cents. How does the clerk determine the change to give you? If the clerk follows a greedy algorithm, he or she gives you two quarters, a dime, and three pennies. That is the smallest number of coins that will equal 63 cents.
```

Comment:
Codacy found an issue: [Split this 322 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6350770)